web/src/ee/features/playground/server/authorizeRequest.ts

```diff
@@ -0,0 +1,25 @@
+import { getIsCloudEnvironment } from "@/src/ee/utils/getIsCloudEnvironment";
+
+import { ApiError, UnauthorizedError, ForbiddenError } from "@langfuse/shared";
+import { getAuthOptions } from "@/src/server/auth";
+import { getServerSession } from "next-auth";
+
+export type AuthorizeRequestResult = {
+  userId: string;
+};
+
+export const authorizeRequestOrThrow = async (
+  projectId: string,
+): Promise<AuthorizeRequestResult> => {
+  if (!getIsCloudEnvironment())
+    throw new ApiError("This endpoint is available in Langfuse cloud only.");
+
+  const authOptions = await getAuthOptions();
+  const session = await getServerSession(authOptions);
```

Comment:
How is the JWT cookie passed from the request in here? Maybe we can use [this](https://github.com/langfuse/langfuse/blob/56e66c195119f0cea0535cd61eafe5d7068b6ff8/web/src/server/auth.ts#L363-L364) here? This is also used in tRPC [here](https://github.com/langfuse/langfuse/blob/56e66c195119f0cea0535cd61eafe5d7068b6ff8/web/src/server/api/trpc.ts#L53)